src/store/pdfFiles/actions.js

```diff
@@ -2,10 +2,12 @@ import { createAction } from 'redux-actions';
 import { JOB_FILE } from '../constants';
 import { setFile } from '../../services/pdfStorage';
 
-const addPdfFile = createAction('PDF_FILE_ADD', ({ file, hasBackup }) => ({
+export const addPdfFile = createAction('PDF_FILE_ADD', (file, hasBackup = false, id, contentMD5) => ({
```

Comment:
when using more than 2-3 arguments it is recommended to wrap them into an object, so I would keep that wrapper